service-front/module/Application/src/Helpers/VoucherMatchLpaActorHelper.php

```diff
@@ -92,9 +93,9 @@ private function compareDob(string $dob, array $actor): bool
      * @param string $firstName
      * @param string $lastName
      * @param string $dob
-     * @return Actor[]
+     * @return false|Actor
```

Comment:
```suggestion
     * @return false|Actor[]
```
Seems it should still be an array here

Reply:
It actually is just a single `Actor` rather than an array of them returned now. I changed this as we only ever surface the first match to users (and its only very rare circumstances where there would be >1 anyways) so its simpler downstream to just return one here.